Open Judge System/Services/OJS.Services/Data/ParticipantScores/ParticipantScoresDataService.cs

```diff
@@ -125,7 +134,8 @@ public void Delete(IEnumerable<ParticipantScore> participantScoresEnumerable)
         public void AddBySubmissionByUsernameAndIsOfficial(
             Submission submission,
             string userName,
-            Participant participant)
+            Participant participant,
+            bool shouldUpdateTotalScoreModifierOn = true)
```

Comment:
Here we should always update the date, as it is new score

Reply:
The logic was changed